OgcApi.Net.MbTiles/MbTilesProvider.cs

```diff
@@ -113,12 +113,17 @@ public async Task<byte[]> GetTileAsync(string collectionId, int tileMatrix, int
             if (tileOptions.MaxZoom.HasValue && tileMatrix > tileOptions.MaxZoom.Value)
                 return null;
 
-            var date = datetime.Substring(0, 10);
-            var fileName = Path.GetFileNameWithoutExtension(tileOptions.FileName) + $"_{date}.mbtiles";
+            string dateString = "";
+            if (datetime != null)
+            {
+                dateString = "_"+ datetime.Substring(0, 10);
```

Comment:
Use DateTimeInterval to parse DateTime intervals:
```C#
var dateTimeInterval = DateTimeInterval.Parse(dateTime);
```
Then convert Start property from DateTime to string with specified format